addons/mail/static/src/models/rtc.js

```diff
@@ -474,7 +473,8 @@ registerModel({
             rtcSession.update({ rtcPeerConnection: insertAndReplace({ peerConnection }) });
             this.messaging.models['RtcDataChannel'].insert({
                 dataChannel,
-                rtcSession: replace(rtcSession),
+                label: dataChannel.label,
+                rtcPeerConnection: replace(rtcSession.rtcPeerConnection),
```

Comment:
At a glance, `label` should be computed field and just alias for `RtcDataChannel/dataChannel/label`...